evm/bridge-token-factory/contracts/BridgeTokenFactory.sol

```diff
@@ -245,10 +245,10 @@ contract BridgeTokenFactory is
     ) internal virtual {}
 
     function claimNativeFee(bytes calldata signatureData, BridgeTypes.ClaimFeePayload memory payload) external {
-        bytes memory borshEncodedNonces = Borsh.encodeUint32(uint32(payload.nonces.length));
+        bytes memory borshEncodedNonces = abi.encodePacked(Borsh.encodeUint32(uint32(payload.nonces.length)));
```

Comment:
why do we need encodePacked?

Reply:
I've changes the borsh encoding api, so it returns fixed bytes now.
Anyway this code will be removed.